medical-practitioner-portal/src/quick-links/quick-links.component.ts

```diff
@@ -0,0 +1,18 @@
+import { ViewportScroller } from '@angular/common';
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-quick-links',
+  standalone: true,
+  imports: [],
+  templateUrl: './quick-links.component.html',
+  styleUrl: './quick-links.component.scss',
+})
+export class QuickLinksComponent {
+  constructor(private viewportScroller: ViewportScroller) {}
+
+  public onClick(event: any, elementId: string): void {
```

Comment:
I believe you can `return false` and then you don't need to deal with $event. https://stackoverflow.com/questions/35723748/angular2-eventemitter-and-preventdefault